lib/core/test/unit/Cardano/Wallet/Api/TypesSpec.hs

```diff
@@ -428,13 +425,6 @@ spec = do
         validateEveryToJSONWithPatternChecker
             match
             (Proxy :: Proxy (Api ('Testnet 0) ApiStakePool))
-        -- NOTE See (ToSchema WalletOrAccountPostData)
```

Comment:
?

Reply:
No longer useful since the upgrade to OpenAPI 3.0